# Service to Get Mocap Message.

---
mocap_frame mf
int64 ret
string msg
